units/initrd-battery-check.service.in

```diff
@@ -12,10 +12,15 @@ Description=Check battery level during early boot
 Documentation=man:initrd-battery-check.service(8)
 DefaultDependencies=no
 AssertPathExists=/etc/initrd-release
+After=plymouth-start.service
 Before=local-fs-pre.target
+Wants=local-fs-pre.target
```

Comment:
I don't think we want this to activate that target

Reply:
Yes, I just added that following `systemd.special(7)`:

```
This means: a consuming service should order itself after these targets (as appropriate), but not pull it in. A providing service should order itself before these targets (as appropriate)
and pull it in (via a Wants= type dependency).
```

Other option could be `Before=local-fs-pre.target local-fs.target`

Reply:
right, but I don't think this qualifies as providing a service for local-fs-pre - also we should have this opt-in for now I think, and maybe change to default-enabled in a future release, just in case